miniapps/performance/makefile

```diff
@@ -98,9 +98,13 @@ MFEM_PERF_CXXFLAGS_gcc_ppc = -mcpu=native -mtune=native\
 MFEM_PERF_CXXFLAGS_xlc = -mcpu=native
 
 # - Clang extra options:
-ifneq ($(MFEM_MACHINE),arm64)
+ifeq ($(MFEM_MACHINE),riscv64)
+  MFEM_PERF_CXXFLAGS_clang += -march=rv64gc
+  else
+    ifneq ($(MFEM_MACHINE),arm64)
   # -march=native is unavailable on clang/ARM64 as of 05/2021: support could be added later.
```

Comment:
Does this still apply? Seems like it works OK for the CMake build.

Reply:
After testing building on riscv64, these lines are not necessary and can be removed, only the modification of CMakeList.txt is necessary.